src/Symfony/Component/Security/Core/Security.php

```diff
@@ -57,4 +59,9 @@ public function getToken(): ?TokenInterface
     {
         return $this->container->get('security.token_storage')->getToken();
     }
+
+    public function getFirewallConfig(Request $request): ?FirewallConfig
+    {
+        return $this->container->get('security.firewall.map')->getFirewallConfig($request);
```

Comment:
The `security.helper` service definition needs to be updated for this service to be available

Reply:
Fixed, thanks!